stac_pydantic/api/landing.py

```diff
@@ -2,7 +2,9 @@
 
 from pydantic import BaseModel, Field
 
-from ..shared import ExtensionTypes, Links
+from stac_pydantic.links import Links
+
+from ..shared import ExtensionTypes
```

Comment:
Is the inconsistency between relative and absolute imports intentional?

Reply:
nope, this is from pycharm auto-refactor.  We should be using absolute refs anywyas.